api/src/test/java/org/openmrs/api/ProgramWorkflowServiceUnitTest.java

```diff
@@ -67,4 +72,15 @@ public void getProgramByName_shouldFailWhenTwoProgramsFoundWithSameName() {
 		pws.setProgramWorkflowDAO(mockDao);
 		pws.getProgramByName("A name");
 	}
+	
+	@Test
+	public void saveProgramme_shouldThrowErrorIfConceptOfProgrammeIsNull() {
```

Comment:
rename `saveProgramme_shouldThrowErrorIfConceptOfProgrammeIsNull` to `saveProgram_shouldFailIfProgramConceptIsNull`